test/calculatorTest.js

```diff
@@ -0,0 +1,28 @@
+const chai = require('chai');
+const { assert } = require('chai');
+const calculator = require('../calculator')
+
+
+describe('Test addition', ()=> {
+    it('Should return addition', ()=>{
+        assert.equal(add(2,2), 4)
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 8.](https://app.codacy.com/gh/bakersen/code-quality-badges/pullRequest?prid=8111319)
```suggestion
    assert.equal(add(2,2), 4)
```